image_utils.py

```diff
@@ -0,0 +1,98 @@
+import cv2
+import numpy as np
+import tensorflow as tf
+from typing import Tuple
+
+
+RGB_DIMENSION_COUNT = 3
+graph = tf.get_default_graph()
+
+
+def smile_detecting(face, model):
+    """ Detecting presence or lack of smile on the face and returning apopriate string """
+    face = face[:, :, 0]
+    mean_image = np.mean(face)
+    std_image = np.std(face)
+    face = (face - mean_image) / std_image
+    face = face.reshape(1, 48, 48, 1)
+    with graph.as_default():
+        prediction = model.predict(face)
+    if prediction[0, 0] >= 0.5:
```

Comment:
Rather return a number that can be visualised in the front-end. Additionally lack of smile is not necessarily sadness.

Reply:
done